frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -160,3 +163,12 @@ export const removeTopologyResourceConnection = (edge: Edge): Promise<any> => {
 
   return removeResourceConnection(sourceObj, targetObj);
 };
+
+export const getSeverityAlertType = (alerts: Alert[]) => {
+  const sortedAlerts = sortMonitoringAlerts(alerts);
+  const severityType = sortedAlerts[0]?.labels.severity as AlertSeverity;
+  return severityType;
+};
+
+export const getFiringAlerts = (alerts: Alert[]) =>
+  _.filter(alerts, (alert) => alert.state.toLowerCase() === AlertStates.Firing);
```

Comment:
No need to convert the state value ```toLowerCase```, as API returns the rule state in lowercase now.